pkg/apis/servicecatalog/validation/broker.go

```diff
@@ -90,3 +90,10 @@ func ValidateBrokerUpdate(new *sc.Broker, old *sc.Broker) field.ErrorList {
 	// allErrs = append(allErrs, validateBrokerStatusUpdate(new, old)...)
 	return allErrs
 }
+
+// ValidateBrokerStatusUpdate checks that when changing from an older broker to a newer broker is okay.
+func ValidateBrokerStatusUpdate(new *sc.Broker, old *sc.Broker) field.ErrorList {
+	allErrs := field.ErrorList{}
+	allErrs = append(allErrs, ValidateBrokerUpdate(new, old)...)
+	return allErrs
+}
```

Comment:
I see it done this way in k8s. I do not understand the status/spec dichotomy enough to know if we'll have different code in here for status vs spec. What I see in other validation functions is much more complicated code that I need more time to understand. @pmorie are there guidelines on what functionality to put in `Validate`? Especially regarding the differences between Status & Spec?